subprojects/core/src/integTest/groovy/org/gradle/api/services/BuildServiceIntegrationTest.groovy

```diff
@@ -17,12 +17,8 @@
 package org.gradle.api.services
 
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
-import org.gradle.integtests.fixtures.InstantExecutionRunner
-import org.gradle.integtests.fixtures.RequiredFeature
-import org.gradle.integtests.fixtures.RequiredFeatures
-import org.junit.runner.RunWith
+import org.gradle.integtests.fixtures.IgnoreWithInstantExecution
 
-@RunWith(InstantExecutionRunner)
```

Comment:
Let's keep the runner attached to this test while we're working on the feature that it covers and while the instant execution suite does not use the in-process executor.

Reply:
Done